doc/magic_cards_notes.md

```diff
@@ -998,9 +998,12 @@ Can emulate MIFARE Classic, Ultralight/NTAG families, 14b UID & App Data
 ### Identify
 ^[Top](#top) ^^[Gen4](#g4top)
 
-👉 **TODO** Tag doesn't get identified correctly by latest Proxmark3 client (it might get mislabeled as MFC Gen2/CUID, Gen3/APDU or NTAG21x Modifiable, depending on configured UID/ATQA/SAK/ATS)
```

Comment:
This information is still relevant,   since the new grn4gtu identification has limitations.

Reply:
re-added the note, with precision on the default password requirement.